Airport Ticket Booking/Airport Ticket Booking/Enums/MessageType.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace AirportTicketBooking.Enums
+{
+    public enum MessageType
+    {
+        Error = ConsoleColor.Red,
```

Comment:
I guess I said I'll explain something here,  a note to explain it in the discussing (running out of time).